packages/expansion-hub/src/ExpansionHub.ts

```diff
@@ -50,8 +50,21 @@ export interface ExpansionHub extends RevHub {
     // DIO
     setDigitalSingleOutput(dioPin: number, value?: boolean): Promise<void>;
     setDigitalAllOutputs(bitPackedField: number): Promise<void>;
+
+    /**
+     * Set a digital pin as input or output.
+     * @param dioPin
```

Comment:
Let's rename `dioPin` to `digitalChannel` for consistency.

Reply:
Renamed